client/src/styles/_typography.scss

```diff
@@ -1,3 +1,7 @@
+html {
+  font-size: calc(0.33vw + 1em);
```

Comment:
Eww... Fontsize based on window size 🤢

Reply:
I heard good things about it but I never tried it in a real project. It avoids changing font size on every breakpoint.

But I'm okay resetting a different font-size depending on the screen size.

Reply:
While I see the benefit of this technique, wouldn't we have to change every sizes (width, height, margins, borders...) to a similar value ? Otherwise it would look weird in some situations ?

I dont know i'm not really sure...

Reply:
Removed it!